packages/atlas/src/components/_video/VideoPlayer/VideoPlayer.styles.ts

```diff
@@ -395,6 +399,41 @@ export const Container = styled.div<ContainerProps>`
     }
   }
 
+  .vjs-text-track-display {
+    > div {
+      margin: 0 !important;
+      font-size: 16px;
+
+      > div {
+        inset: unset !important;
+        transition: bottom;
+        transition-delay: ${TRANSITION_DELAY};
+        transition-duration: 200ms;
+        transition-timing-function: ${transitions.easing};
+        padding: 0 ${sizes(3)};
+
+        > div {
+          display: inline-block !important;
+          background-color: ${cVar('colorBackgroundOverlay')} !important;
+          font: ${cVar('typographyDesktopT300Strong')} !important;
+          font-size: 12px !important;
+          letter-spacing: ${cVar('typographyDesktopT300StrongLetterSpacing')} !important;
+          text-transform: ${cVar('typographyDesktopT300StrongTextTransform')} !important;
+          padding: ${sizes(1)} ${sizes(2)};
```

Comment:
I think we would need to do some changes to the padding: 
This is a fullscreen desktop: 
![image](https://user-images.githubusercontent.com/51168865/185555344-eccb4e1c-d8d1-4e31-b2d0-52eaf89caa56.png)

Reply:
Correct me if I'm wrong, but line height also seems wrong on mobile: 
![image](https://user-images.githubusercontent.com/51168865/185557338-684740cb-a78f-4f40-bea1-2a27bf30c10a.png)